openedx/openedx-platform
# SE-3243 Remove injected edxnotes from AboutBlock

This is required to stop get_course_about_section html content from using the edxnotes version of get_html, which injects the edxnotes html into it. We don't want this injected into the html here, because these are snippets meant to be displayed in original form on the course about page.

This fixes a regression introduced by https://github.com/edx/edx-platform/pull/20954

**JIRA tickets**: [OSPR-4952](https://openedx.atlassian.net/browse/OSPR-4952)

**Dependencies**: None

**Sandbox URL**: TBD - sandbox is being provisioned.

**Merge deadline**: "None"

**Testing instructions**:

1. log in
1. set Enable Student Notes to true in the course advanced settings
1. set a non-empty 'hours of effort per week' on the schedule and details settings for the course
1. visit the course about page
1. verify that the effort field is displayed correctly
1. view source and verify that you don't see any instances of edx-notes-wrapper-

**Author notes and concerns**:

This commit also contains some WIP tests (which are not working yet). I can't yet work out how to set an effort, etc. for the course in the test.


**Reviewers**

**Settings**
```yaml
EDXAPP_FEATURES:
  ENABLE_EDXNOTES: true
```
